src/modules/public.ts

```diff
@@ -230,6 +231,23 @@ export default class Public {
     );
   }
 
+  /**
+   * @description get retroactive mining rewards for an ethereum address
+   *
+   * @param ethereumAddress An Ethereum address
```

Comment:
description of this field could be more specific

Reply:
done